tests/cnf/core/network/metallb/tests/bfd-test.go

```diff
@@ -526,8 +525,12 @@ func buildRoutesMap(podList []*pod.Builder, nextHopList []string) (map[string]st
 
 	routesMap := make(map[string]string)
 
-	for num, pod := range podList {
-		routesMap[pod.Definition.Spec.NodeName] = nextHopList[num]
+	for _, pod := range podList {
+		if pod.Definition.Spec.NodeName == "worker-0" {
+			routesMap[pod.Definition.Spec.NodeName] = nextHopList[1]
+		} else {
+			routesMap[pod.Definition.Spec.NodeName] = nextHopList[0]
+		}
 	}
 
```

Comment:
Does this loop need coverage for 3 worker cluster ?
I assume this covers when frr pods are only on worker-0 and worker-1.

Reply:
I returned the func to its orginal form and created a new one for frrk8 test cases. There  I only will use two worker nodes each with one hub pod. Because of security policies it is not possible on the same two communicate between two pods on the same IP address range. So the static routes need to be crossed between the two workers.